Eastern-Research-Group/mywaterway
# Feature/775 update 404 return

## Related Issues:
* [HMW-775](https://jira.epa.gov/browse/HMW-775)

## Main Changes:
* Updated the server to return a json based error message for non-existent routes under `api/*`. Other non-existent routes still return the 404 html file.
  * Brad and I were doing some testing earlier in HMW and decided that HMW and EQ should both return a json based error under the `api/*` path.
* Renamed `400.html` to `404.html`. 404 just makes more sense, since we are trying to catch non-existent routes. 

## Steps To Test:
1. Open dev tools console
2. Navigate to http://localhost:3000/community/dc/overview
3. Verify the app loads
4. Navigate to http://localhost:3000/bogusRoute/
5. Verify you get a `Sorry, but this web page does not exist.` message and the status code is 404
6. Navigate to http://localhost:9090/bogusRoute/
7. Verify you get an html file with the error
8. Navigate to http://localhost:9090/api/bogusRoute/
9. Verify you get a json error message
